<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="LAPTOP-IP1C4S1G" failures="1" tests="3" name="Test(failed)(failed)" time="0.112" errors="0" timestamp="2023-04-05T18:29:42 IST">
  <testcase classname="TestNG.sample2$sample1" name="TC6" time="0.026">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at TestNG.sample2$sample1.TC6(sample2.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- TC6 -->
  <testcase classname="TestNG.sample3$sample1" name="TC10" time="0.003"/>
  <testcase classname="TestNG.sample3$sample1" name="TC9" time="0.008"/>
  <testcase classname="TestNG.sample3$sample1" name="TC8" time="0.0">
    <ignored/>
  </testcase> <!-- TC8 -->
  <testcase classname="TestNG.sample2$sample1" name="TC7" time="0.0">
    <ignored/>
  </testcase> <!-- TC7 -->
  <testcase classname="TestNG.sample2$sample1" name="TC4" time="0.0">
    <ignored/>
  </testcase> <!-- TC4 -->
  <testcase classname="TestNG.sample2$sample1" name="TC5" time="0.0">
    <ignored/>
  </testcase> <!-- TC5 -->
</testsuite> <!-- Test(failed)(failed) -->
